build.gradle

```diff
@@ -107,6 +107,14 @@ allprojects {
                 throw new RuntimeException("Invalid corretto.debug_level")
         }
 
+        def freetypeIncludePath = project.hasProperty("corretto.freetype_include") ? project.findProperty("corretto.freetype_include") : null
```

Comment:
findProperty(string) will return null if it's not found. so all of these hasProperty checks can probably just be removed and changed to find since you have the checks for null right after.

Reply:
Good catch. I was thinking it was an exception, but that's `getProperty`.